apps/web/src/components/IntervieweeInfo/IntervieweeInfo.tsx

```diff
@@ -0,0 +1,109 @@
+import { useCallback } from 'react';
+import Webcam from 'react-webcam';
+import IconButton from '@mui/material/IconButton';
+import PhotoCamera from '@mui/icons-material/PhotoCamera';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useRef, useState } from 'react';
+import styles from '../IntervieweeInfo/index.module.css';
+
+type Info = {
+  fullName: string;
+  email: string;
+  phone: string;
+  dateOfBirth: string;
+  workingStatus: 'Učenik' | 'Student' | 'Nezaposlen' | 'Zaposlen';
+  institutionName: string;
+  yearOfStudy: number;
+  field: 'Dev' | 'Design' | 'Marketing' | 'Multimedija';
+  referral:
+    | 'Društvenih mreža'
+    | 'Predstavljanja na fakultetima/školama'
+    | 'Medija'
+    | 'Prijatelja ili poznanika'
+    | 'Ostalo';
```

Comment:
`referral`  i `workingStatus` bi prebacia u obicni string jer nam nije bitno koje su vijednosti tocno a daje nam na fleksibilnosti ako prominimo input

za `field`  (to jest sad se zove discipline) bi trebalo koristit taj tip (`Discipline`) umjesto liste vrijednosti